x/oracle/types/keys.go

```diff
@@ -61,33 +61,30 @@ func GetAggregateExchangeRateVoteKey(v sdk.ValAddress) (key []byte) {
 	return append(key, address.MustLengthPrefix(v)...)
 }
 
-// GetMedianKey - stored by *denom* and *block*
-func GetMedianKey(denom string, blockNum uint64) (key []byte) {
+// MedianKey - stored by *denom* and *block*
+func MedianKey(denom string, blockNum uint64) (key []byte) {
```

Comment:
renamed, because we construct a value rather then extracting it.

Reply:
Lets do this same renaming for the other oracle key constructor functions in another pr 👍